src/differential_arithmetic.jl

```diff
@@ -53,7 +53,7 @@ end
 
 Base.:+(a::One, b::One) = extern(a) + extern(b)
 Base.:*(::One, ::One) = One()
-for T in (:AbstractThunk, :Any)
+for T in (:AbstractThunk, :Composite, :Any)
     @eval Base.:+(a::One, b::$T) = extern(a) + b
```

Comment:
Does this even make sense for `Composite`?
What is `one for a composite?

Reply:
Yes, you are right, addition with `One()` doesn't make any sense. I'll fix that